### DISCUSSION TITLE
Enforce i18 subpath routing for default locale

### DISCUSSION THREAD

**Discussion by dmudro** [vercel/next.js#20677] [Help]
Let's consider the [textbook example](https://nextjs.org/docs/advanced-features/i18n-routing#sub-path-routing):

```
// next.config.js
module.exports = {
  i18n: {
    locales: ['en', 'fr', 'nl-NL'],
    defaultLocale: 'en-US',
  },
}
```

So for `pages/blog.js` the URL paths would be:

- `/blog`
- `/fr/blog`
- `/nl-nl/blog`

Now I want to have `en` in subpath for default locale too: `/en/blog` and have it **enforced**. In other words there would always be a locale subpath:
- `https://example.com/en/...`
- `https://example.com/fr/...`
- `https://example.com/nl-nl/...`

What are my options using the latest Next.js (10.0.4 at the time of writing)?

**Comment by michrome** [comment]
It might be easiest to stick with `/` for now then adopting something like https://github.com/vercel/next.js/pull/19035 if it's merged into a future version.

**Reply by dmudro** [reply]
Many thanks for the PR

**Comment by dmudro** [comment]
Also in this summary: https://github.com/vercel/next.js/discussions/20240
